apps/aecore/lib/aecore/channel/channel_state_peer.ex

```diff
@@ -293,16 +295,17 @@ defmodule Aecore.Channel.ChannelStatePeer do
           ChannelTransaction.onchain_tx_payload()
         ) :: DataTx.t()
   defp initialize_new_mutual_onchain_tx(
-         %ChannelStatePeer{
-           initiator_pubkey: initiator_pubkey,
-           responder_pubkey: responder_pubkey
-         },
+         %ChannelStatePeer{} = peer_state,
          fee,
          nonce,
          type,
          spec
        ) do
-    DataTx.init(type, spec, [initiator_pubkey, responder_pubkey], fee, nonce)
+    if type.chainstate_senders? do
```

Comment:
What does chainstate_senders? mean?

Reply:
This was introduced by Channel PR, becouse some transactions don't contain senders in encoding.

Reply:
Some transactions do not contain sender information. &chainstate_senders/0 checks whether we need to consult the chain in order to obtain sender information. These transactions must have their senders set to an empty list.

Reply:
Ok, thanks